README.md

```diff
@@ -1,25 +1,70 @@
+providing connectivity for the sensors.
 
 ---
 
 ## Environment for Weather Station project  
-The complete IoT Environment is builded with following components:
-* LWM2M cloud server  
-* Ci40 application which allows usage of clicks in mirkroBUS sockets.
-* Contiki based applications build for clicker platform:
-  *  [Temperature sensor](https://github.com/CreatorKit/temperature-sensor)
-* a mobile application to present Weather measurements.  
+The complete IoT Environment is built using the following components:
+* IoT Framework
+* Ci40 Application - Allows the usage of clicks in mirkroBUS sockets
+* Contiki based applications - build for 6LoWPAN clicker platform:
+  *  [Temperature Sensor](https://github.com/CreatorKit/temperature-sensor)
+* Mobile Application - Presents weather measurements  
 
-## Ci40 Application Specific Dependencies
-This application uses the [Awa LightweightM2M](https://github.com/FlowM2M/AwaLWM2M) implementation of the OMA Lightweight M2M protocol to provide a secure and standards compliant device management solution without the need for an intimate knowledge of M2M protocols. Additionnaly MikroE Clicks support is done through [LetMeCreate library](https://github.com/CreatorDev/LetMeCreate).
+## Application Dependencies
 
-## Integration With OpenWrt
-It's assumed that you have build envriroment for ci40 openWrt described [here](https://github.com/CreatorKit/build) and this is located in folder `work`. So structure inside will be:
+The Weather Station uses [Awa LwM2M](https://github.com/FlowM2M/AwaLWM2M), an 
+implementation of OMA Lightweight M2M protocol. Awa provides a secure and standard 
+compliant device management solution, without the need for an intimate knowledge of 
+M2M protocols.  
+The MikroE Clicks (sensors) use
+[LetMeCreate](https://github.com/CreatorDev/LetMeCreate), an open source library 
+design to speed up the development with Ci40.
+
+
+### How to Install Dependencies on Ci40
+
+AwaLWM2M and LetMeCreate have a package ready to install on OpneWRT. For this, 
+on your Ci40 console execute:
+
+```bash
+opkg update 
+opkg install awalwm2m
+opkg install letmecreate 
+```
+
+## How to Install the Application
+
+To install the Weather Station, you can use the prebuild package (quick and easy),
+or you can build from the source code, if you plan to edit source (advanced users).
+
+### Install the Pre-built Package
+
+This quick process consists in transfer the latest release .ipk package file 
+onto Ci40, for example using **WGET**.
+Then on the package directory, to complete the installation execute:
+```bash
+opkg install package_name
+```
+
+### Building From Source
+
+This process fits for the most developers who want to edit and build from the 
+source code, confortable setting the building environment for OpenWRT applications. 
+
+It's assumed that you have build envriroment for ci40 openWrt described 
```

Comment:
I think we should stick with one name for Ci40 board. We usually use uppercase "C" and lowrcase "i"